code/src/components/Loading.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { RotatingLines } from 'react-loader-spinner'
+
+const Loading = () => {
+  return (
+    <div className="loading-screen">
+      <RotatingLines
+        visible
+        height="80"
+        width="80"
+        ariaLabel="blocks-loading"
+        wrapperStyle={{}}
+        wrapperClass="blocks-wrapper"
+        strokeColor="grey" />
+    </div>
+  )
+}
+
```

Comment:
I did not spend time to actually make this page to be properly coded, makes me look a bit sloppy in comparison to yours :) Well done